RELEASE_NOTES.md

```diff
@@ -1,9 +1,40 @@
-# Cosmos SDK v0.47.0-beta1 Release Notes
+# Cosmos SDK v0.47.0-rc1 Release Notes
 
-There is no release notes for pre-releases. 
+Cosmos SDK `v0.47.0-rc1` contains all the features and changes that are planned for the final v0.47.0 release.
+This release candidate is intended to give application developers and validator operators a chance to test the release candidate before the final release.
+
+The main changes in this release are:
+
+* Upgrade to Tendermint [v0.37.0](https://github.com/tendermint/tendermint/blob/v0.37.0-rc2/CHANGELOG_PENDING.md).
+    * With the notable introduction of [ABCI 1.0](https://medium.com/the-interchain-foundation/tendermints-new-application-blockchain-interface-abci-86d46bd6f987).
+    * Changes of events keys and values from `[]byte` to `string`.
+
+* Support of [ABCI 1.0 in the SDK](https://docs.cosmos.network/main/building-apps/app-mempool).
+    * Allows chains to set their own mempool implementation.
+    * Support of the new `PrepareProposal` and `ProcessProposal` ABCI methods.
+
+* Deprecation of `x/params` modules.
+    * Modules params are now handled directly by the modules themselves, with the message `MsgUpdateParams`.
+    * All core modules have migrated away from `x/params`. It is recommended to migrate your custom modules as well.
+
+* Migration from `gogo/protobuf` to `cosmos/gogoproto`.
+    * The SDK was using the now unmaintained `gogo/protobuf` library. This has been replaced by [`cosmos/gogoproto`](https://github.com/cosmos/gogoproto) which is a fork of `gogo/protobuf` with some improvements and fixes, that is maintained by the Cosmos SDK team.
+    * This change is not transparent for applications developers. All proto files should be regenerated with the new library.
```

Comment:
Should we say that teams only need to use x version of the docker file?

Reply:
Yes, this is present in the UPGRADING.md, précising here too.